pkg/operation/botanist/component/vpa/updater.go

```diff
@@ -23,7 +23,6 @@ import (
 	"github.com/gardener/gardener/pkg/operation/botanist/component"
 	"github.com/gardener/gardener/pkg/utils"
 	gutil "github.com/gardener/gardener/pkg/utils/gardener"
-
```

Comment:
Nit: You could revert this change.

Reply:
The change was made by some of the tools in GoLand's automatic formatting pipeline. Do we have an established convention about how we organise imports? I'd like to configure my tooling accordingly, otherwise I'll be running into the same issue ad infinitum.